src/main/java/uk/gov/justice/hmpps/datacompliance/events/publishers/sqs/DataComplianceNoOpEventPusher.java

```diff
@@ -49,6 +51,12 @@ public void requestFreeTextMoratoriumCheck(final OffenderNumber offenderNo,
                 offenderNo.getOffenderNumber(), retentionCheckId);
     }
 
+    @Override
+    public void requestOffenderRestrictionCheck(OffenderNumber offenderNumber, Long retentionCheckId, Set<OffenderRestrictionCode> offenderRestrictionCode, String regex) {
```

Comment:
Really minor but for consistency it would be good to keep the finals on input parameters

Reply:
updated